lib/page/controller/page.summary/page.summary.controller.js

```diff
@@ -122,19 +122,25 @@ SummaryController.setContents = (pageInstance, userData, res, skipActions) => {
           }
         }
       }
-      if (stepInstance.sectionHeading) {
-        if (stepInstance.sectionHeading !== previousInstance.sectionHeading) {
+
+      const sectionHeading = stepInstance.sectionHeadingSummary || stepInstance.sectionHeading
```

Comment:
Maybe there should be a `getInstanceSectionHeading` method?

But not right now - `getInstanceProperty` can be passed an array of properties to check

getInstanceProperty(stepInstance._id, ['sectionHeadingSummary', 'sectionHeading'])

Reply:
ok for now have used a `getInstanceProperty`